server/issue.go

```diff
@@ -73,6 +75,37 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		create.Fields.Description = permalink
 	}
 
+	rootId := create.PostId
+	parentId := ""
+	if post.ParentId != "" {
+		// the original post was a reply
+		rootId = post.RootId
+		parentId = create.PostId
+	}
+
+	if len(create.RequiredFieldsNotCovered) > 0 {
+
+		v, _ := query.Values(create.Fields)
+
+		message := "The project you tried to create an issue for has **required fields** this plugin does not yet support. "
```

Comment:
Since we are sending the fields not covered up, why not tell them which fields where not covered?

Reply:
I thought about doing that and sent the values instead of true/false for this reason.  Turns out the linked Jira page states at the top which fields are required (that weren't handled).  So I was debating where to add the fields to the post.   How about I cut the PR non-drafted PR and then add this feature.  I'm still working out some issues. 

Another good reason to add this enhancement would be so users have a traceable message listing fields we currently don't handle and they can propagate that info up to management and us.

Reply:
@jasonblais This is a good request from @crspeller.  I'm curious what you would like the final message to look like?
> The project you tried to create an issue for has **required fields** this plugin does not yet support. 
> Please [create your Jira issue manually](permalink)
> Unsupported Fields:
>   - FieldName 1
>  - FieldName 2

Reply:
If this is a low risk change, I'm all for it. In this case we can bring the action first, so the list of unsupported fields flows naturally from the second sentence.

```
[Please create your Jira issue manually](permalink). The project you tried to create an issue for has these **required fields** the plugin does not yet support:
- FieldName 1
- FieldName 2
- ...
```

/cc @asaadmahmood if you have feedback on the above proposed message

Reply:
Looks fine for me, especially if its just a concern case, which we may in the future would further reduce (by supporting additional fields too).

Reply:
@jfrerich This can also be filed as a separate improvement for later. The only blocking change from PM side is turning the message to be ephemeral

Reply:
Thanks for the feedback, J.  I'm finishing my RC testing right now and will get this done right after.